tslearn/datasets.py

```diff
@@ -296,3 +331,13 @@ def load_dataset(self, dataset_name):
         y_train = npzfile["y_train"]
         y_test = npzfile["y_test"]
         return X_train, y_train, X_test, y_test
+
+if __name__ == "__main__":
```

Comment:
Are there still failures on your side (eg because of files not available)?

Reply:
All datasets are loading correctly. I tested it on a new machine without cached datasets.

In future, if remote files change (i.e., UCR zip archives, or http://www.timeseriesclassification.com/singleTrainTest.csv) the loading may fail again, though.